index.html

```diff
@@ -59,6 +59,46 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        As the popularity of devices with immersive capabilities continues to
+        expand, there comes an increasing desire to make use of them to consume
+        three-dimensional content across a range of use cases - and a spectrum
+        of complexity. With that in mind, it is necessary to augment the web
+        with appropriate standards to meet those needs at every point along the
+        spectrum. While solutions like [[webxr]] are necessary for more complex
+        experiences in this regard, the level of information disclosure
+        required for such a session should not be necessary for the basic,
+        spatial display of an object in the context of a page. This is the area
+        of focus for the HTML `&lt;model&gt;` element.
+      </p>
+      <p>
+        There are a variety of solutions for displaying three-dimensional
+        models on the web, including general WebGL libraries and frameworks
+        that have the ability to parse and render models. There are also more
+        purpose-built libraries devoted to a simpler set of capabilities around
+        a single model, providing a simpler API to achieve this. In both cases,
+        however, these libraries exist as a layer of abstraction on top of
+        browser APIs. As such their presentation of capability cannot
+        constitute as stable a basis for browser- and system-level
+        accessibility features.
```

Comment:
Is there any basis for this statement? It's very normal for developers to build sites on frameworks and libraries that expose higher-level control of the browser- and system-level a11y features. `<model-viewer>` is an example of a 3D library with solid, customizable a11y features, e.g. https://modelviewer.dev/examples/loading/#useA11y

Reply:
It's common but not bedrock. I absolutely agree that `<model-viewer>` is an excellent citizen, but there's a difference in kind between a11y features that are intrinsically part of a standard specification, and features that are exposed by convention. 

I'd also contend that most other JS frameworks providing a11y additions aren't doing the same level of conversion from browser-facing implementation to user-facing experience, i.e. text-to-text, DOM-to-buttons, rather than canvas-to-3D model.

Reply:
>but there's a difference in kind between a11y features that are intrinsically part of a standard specification, and features that are exposed by convention.

That's true: the standard is slow and the conventions are agile. In my various a11y reviews it's clear that almost no one has thought about what a11y means in 3D. We have done a number of things to make it better, but I think this is a very emerging discipline. The browser's built-in aria system provides *some* benefit, but I think much more would come from designing high-contrast shaders and such. However, I don't know of anyone even prototyping such a thing yet, much less doing a11y user research. Do you feel comfortable standardizing 3D a11y in such a state?

Reply:
The document should justify the approach and compare and contrast to the demonstrated standards already in this space and described by W3C Community Groups (and Class A Liaisons)... specifically the ISO-IEC Standards of X3D. With a nice alignment to the DOM, CSS, HTML, glTF2, and WebAudio, X3D 
Version 4 was just ratified through the full standard process and voting by  National Bodies: 

https://web3d.org/x3d4 
see also: https://webx3d.org/

The standard is implemented by two open source javascript  libraries:
 X3DOM https://www.x3dom.org/   (WebXR supported)
 X_ITE https://github.com/create3000/x_ite/wiki 

The MIME types for X3D are model/x3d+xml, model/x3d+binary, and model/x3d+vrml. The Internet Assigned Numbers Authority (IANA) has approved model/x3d+xml as the primary MIME media type for X3D media in XML encoding.

In the Metaverse Standards Forum 3D Web Interoperability Group, we are discussing the  X3D pattern of of url and uris for different kinds of linked content and experiences; also we are looking at the conformance mechanism inherent in the Profiles and Components that enables different levels of interactive 3D functionality to be specified.

Reply:
We believe that we should work together to update this spec

https://www.rfc-editor.org/rfc/rfc2077.html